Benchmarks/Benchmarks/Predicates/BenchmarkPredicates.swift

```diff
@@ -67,6 +67,8 @@ func registerPredicateTests_disabled() {
 #endif
 
 func registerPredicateTests() {
+    // Temporarily disabled due to link issue
+    #if FOUNDATION_FRAMEWORK
```

Comment:
Should this be `#if !USE_PACKAGE && !os(macOS)` (the link issue was only present in non-Darwin toolchains).

FWIW, the link issue should now be resolved, but the macro doesn't exist in a Windows toolchain yet so eventually this will have to be `#if USE_PACKAGE || !os(Windows)`